EIPS/eip-7201.md

```diff
@@ -4,7 +4,8 @@ title: Namespaced Storage Layout
 description: A formula for the storage location of structs in the namespaced storage pattern.
 author: Francisco Giordano (@frangio), Hadrien Croubois (@Amxx), Ernesto García (@ernestognw), Eric Lau (@ericglau)
 discussions-to: https://ethereum-magicians.org/t/eip-7201-namespaced-storage-layout/14796
-status: Draft
+status: Last Call
+last-call-deadline: 2023-09-15
```

Comment:
Generally proposals go from Draft to Review, then to Last Call. Any particular reason you're skipping Review?

Reply:
The EIP has been effectively in Review for months now. I recognize we should've formalized that in the status but I only realized once we wanted to move to Last Call.